docs/gen-ai/gen-ai-spans.md

```diff
@@ -20,8 +21,11 @@ A request to an Generative AI is modeled as a span in a trace.
 
 **Span kind:** MUST always be `CLIENT`.
 
-The **span name** SHOULD be set to a low cardinality value describing an operation made to an Generative AI system.
-For example, the API name such as [Create chat completion](https://platform.openai.com/docs/api-reference/chat/create) could be represented as `completion gpt-4` to include the API and the model name.
+## Name
+
+GenAI spans MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.33.0/specification/trace/api.md#span).
+The **span name** SHOULD be `{gen_ai.operation.name} {gen_ai.request.model}`.
+Semantic conventions for individual GenAI systems MAY specify different span name format.
```

Comment:
Customers I've worked with want to add a meaningful span name here based on their workflow (some workflow engines like LangChain and Promptflow may do this too). In the context of the application, its low cardinality. For example, "Summarize" instead of "Completions" to differentiate from "Generate response".  This may be in a trace flow such as `Embeddings` -> `Retrieval` -> `Summarize` -> `Generate response`. The last two both being Completions (or Chat) operations. 

Should we be clearer that this is OK?

Reply:
would users instrument such calls themselves? 
I.e. is the ask to allow it more explicitly or to change the operation names for everyone?

Reply:
Hmmm. I think it would be good for more domain-specific instrumentation (like using specific langchain or llamaindex abstractions) to have a generic-ish name they can use if that's the abstraction they're providing. We could offer guidance on what that name should be, and ask that it's a verb name of low cardinality, and list some examples like above.

Reply:
I think a library (langchain, etc) may want to provide a default that is more descriptive than Operation Name and Model. And a user may want to override it with a domain specific name, but the guidance should remain that it is low cardinality.

Reply:
I'm updating this sentence to be 
> Semantic conventions for individual GenAI systems and frameworks MAY specify different span name format.

Do you feel it gives enough freedom to use a different pattern? Or is there something different you'd like to have there?

The cardinality requirements are part of the general guidelines for span name linked on line 26.

Reply:
Yes, I think that's enough to support the requirement.